pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -208,6 +209,12 @@ func setDefaultLogsConfig(c *latestV1.SkaffoldConfig) {
 	}
 }
 
+func setDefaultKubectlDryRun(c *latestV1.SkaffoldConfig) {
+	if c.Deploy.KubectlDeploy != nil && c.Deploy.KubectlDeploy.DryRun == "" {
+		c.Deploy.KubectlDeploy.DryRun = "client"
+	}
+}
```

Comment:
Do we need this? You're already handling `""` in the CLI code itself, and it provides false comfort since you're also passing in `""` explicitly from the Kustomize code.

Reply:
that's true, it doesn't look like this is needed. I'll remove this